recipes/pagmo/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.0" %}
+
+package:
+  name: pagmo
+  version: {{ version }}
+
+source:
+  url: https://github.com/esa/pagmo2/archive/v{{ version }}.tar.gz
+  sha256: 022af6bad2f1d083c2cacf4b121722541df80b6f168519a8eba0709cef02163a
+
+build:
+  number: 0
+  skip: true  # [(win and py27) or win32]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - eigen
+    - nlopt
+    - boost-cpp 1.63.*
+  run:
+    - nlopt
```

Comment:
Yes, we use the header only bits of Boost and eigen. Should I declare them as run deps as well?